launch.py

```
@@ -253,6 +300,12 @@ def activate_conda_env(env_name: str = None) -> bool:
     """Activate a conda environment."""
     env_name = env_name or CONDA_ENV_NAME
 
+    # Validate environment name to prevent command injection
+    import re
```

Ruff (0.14.1)

Redefinition of unused `re` from line 19

Remove definition: `re`

(F811)